### ISSUE TITLE
not working as intended, nothing happens when u click

### ISSUE THREAD

**Issue by Ming-A** [shlagevuk/docker-hakuneko#5]
![image](https://user-images.githubusercontent.com/41011175/107119877-57740780-68c5-11eb-8c83-bcaf8e6bfcb6.png)

**Comment by ghost** [comment]
Yeah the "Developer" of this prolly followed some beginner Docker tutorial and threw this mess of a project together in 2 minutes.
